packages/esm-patient-chart-app/src/index.ts

```diff
@@ -29,19 +30,20 @@ function setupOpenMRS() {
   defineExtensionConfigSchema('nav-group', genericNavGroupConfigSchema);
   defineExtensionConfigSchema('dashboard', genericDashboardConfigSchema);
 
+  const t = i18next.getFixedT(null, moduleName);
   registerBreadcrumbs([
     {
       path: spaBasePath,
-      title: 'Patient',
+      title: () => Promise.resolve(t('patientBreadcrumb', 'Patient')),
       parent: `${window.spaBase}/home`,
     },
     {
```

Comment:
We probably need some comments here, at least to cover the standard dashboards, e.g.

```ts
/**
 * t('visitDashboard', 'Visit dashboard');
 */
```

(Not sure if that's a real one.